CONTRIBUTING.md

```diff
@@ -179,17 +176,7 @@ var/count = 0
 var/c = 1
 ```
 ***
-Avoid digits in names.
```

Comment:
Digits **should** be avoided in names. As having a millions things names var/text1, var/text2, var/text3 makes code borderline unreadable. This is also good sound coding advise that you are removing. Please do not encourage bad coding styles and practices.

Reply:
This doesn't encourage using numbers everywhere, its still adviseable to have descriptive variable names. But there are many cases where you have multiple objects of an identical purpose, and calling them thing1 and thing2 is not functionally different from thingA and thingB, or thing_one and thing_two

There already is a rule, elsewhere in this file, about having properly descriptive names.  That fulfills what you're asking this rule to.

This rule was just specifically and arbitrarily excluding digits from being used in situations where they were perfectly valid